operations/cancellationpolicies.md

```diff
@@ -102,6 +100,197 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `RelativeFee` | number | required | Relative value of the fee, as a percentage of the reservation price. |
 | `IsActive` | boolean | required | Whether the cancellation policy is still active. |
 
+## Get cancellation policies
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Given the Reservation interval and Rates, returns the Cancellation Policies that are currently applicable per each rate.
+
+### Request
+
+`[PlatformAddress]/api/connector/v2/cancellationPolicies/get`
```

Comment:
```suggestion
`[PlatformAddress]/api/connector/v1/cancellationPolicies/getAll/2024-27-08`
```
We should not be touching the v1 if we want to create a new version of the endpoint, this is reserved for the whole connector API version. Which would have new authentication, follow proper RESTful principals etc. 

Also, we use getAll as a standard retrieval operation.

Reply:
(watch out for that ISO date though)